### DISCUSSION TITLE
[FEATURE REQUEST] Simulated Input Delay for Training or Offline modes

### DISCUSSION THREAD

**Discussion by RagingRowen** [ikemen-engine/Ikemen-GO#880] [Feature Request]
Certain modern FGs (Can't name them all) have this feature implemented in their training modes where you can adjust how many delay frames you want.

This could surely help prepare for online matches where the connection isn't up to no good.

**Comment by potsmugen** [comment]
I do think this should exist because rollback has the input delay option. It makes sense to play/practice locally with the same delay you have online.
